smartsim/_core/control/controller.py

```diff
@@ -538,6 +567,7 @@ def _launch_step(
         # that has ran and completed, relaunch the entity.
         elif completed_job is not None and completed_job.entity is entity:
             try:
+                self.symlink(job_step, entity)
```

Comment:
Same thoughts as in above comment.

Reply:
Handled in the same way as mentioned above.

Reply:
Well done!